refarch-integrations/pom.xml

```diff
@@ -31,13 +31,15 @@
 
         <!-- Core Frameworks -->
         <spring-cloud.version>2023.0.3</spring-cloud.version>
+        <refarch-tools.version>1.0.1</refarch-tools.version>
```

Comment:
I would this property declaration down to the utitlity section to be in sync with refarch-templates repository.

Reply:
Done